<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="graphPlotter">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>601</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopPlot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPlot">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPlot">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QComboBox" name="signalDrop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Select the type of signal :</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Select suitable frequency :
</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="freqInput">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGraphicsView</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
